src/fool_ai_detector/service/fake_generator_picture.py

```diff
@@ -0,0 +1,31 @@
+import skimage
+import os
+import random
+
+from fool_ai_detector.model.generator import Generator
+
+
+class FakeGeneratorPicture(Generator):
+    base_path_this_class = os.path.dirname(os.path.abspath(__file__))
+    dir_path = os.path.join(base_path_this_class, '..', 'resources', 'ai_gen_images')
+    images = []
+
+    def __init__(self):
+        for image in os.listdir(self.dir_path):
+            if os.path.isfile(os.path.join(self.dir_path, image)):
+                self.images.append(os.path.join(self.dir_path, image))
+
+    def generate(self, output_file_path):
+        """
+           Method that takes one random image of the ai_gen_image directory and puts it in another directory.
+        """
```

Comment:
I know like this. Is there a reason why you added another tab?
```suggestion
        """
        Method that takes one random image of the ai_gen_image directory and puts it in another directory.
        """
```